sfxclient/httpsink.go

```diff
@@ -92,7 +92,8 @@ func (h *HTTPSink) handleResponse(resp *http.Response, respValidator responseVal
 		return errors.Annotate(err, "cannot fully read response body")
 	}
 
-	if resp.StatusCode != http.StatusOK {
+	// all 2XXs
+	if resp.StatusCode < http.StatusOK || resp.StatusCode > http.StatusIMUsed{
```

Comment:
No space before brace is passing `go fmt` ? Also to be more future proof this probably should be:

```go
if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
    ...
}
```